pkg/plugins/avagoConfig.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package plugins
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/ava-labs/avalanche-cli/pkg/application"
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+)
+
+func EditConfigFile(app *application.Avalanche, subnetID string, networkID string, configFile string, forceWrite bool) error {
```

Comment:
probably add a comment to the function, also telling that the function creates a config file if it does not exists

Reply:
done